src/modules/OnlineUpdateDialogs.rb

```diff
@@ -553,7 +547,6 @@ def show_eol_products(products)
       )
 
       UI.OpenDialog(dialog_description)
-      user_ret = UI.UserInput
```

Comment:
This is wrong.
The result is unused because we don't care which button is used to close the popup. We still need to wait for the user to press it after reading the popup.

Reply:
right. I've put the UserInput back